src/lib.rs

```diff
@@ -970,8 +999,9 @@ impl Wizer {
     }
 }
 
-/// A "diff" of Wasm state from its default value after having been initialized.
-struct Diff<'a> {
+/// A "snapshot" of Wasm state from its default value after having been
```

Comment:
s/from its default value//?

Reply:
Gah this is what I get for blind replacement! Thanks!